src/animation/interpolants/SmoothInterpolant.js

```diff
@@ -0,0 +1,103 @@
+/**
+ *
+ * Cubic hermite spline interpolant.
+ *
+ *
+ * @author tschw
+ */
+
+THREE.SmoothInterpolant = function( times, values, stride, result ) {
+
+	THREE.Interpolant.call( this, times, values, stride, result );
+
+};
+
+Object.assign( THREE.SmoothInterpolant.prototype, THREE.Interpolant.prototype, {
+
+	constructor: THREE.SmoothInterpolant,
+
+	DefaultParameters: {
```

Comment:
> I do know of Interpolators with a lot of parameters, but they are generally not used in real-time contexts -- like the bias/continuity/tensition smooth interpolator,

I don't remember the names of the inventors, but these are the Lightwave / Max splines :-).

> Interesting to have the velocities.

Which velocities do you mean? The values you are referring to are Boolean ones, that specify how the spline behaves at the ends when running out of keyframes. These are needed to implement proper looping (which I haven't done yet, but `.parameters` is defined as an external structure, so it can be shared and we don't have to tell every single track that the loop count has reached zero and we're going to stop, for example).

Inside a loop, we want a "natural spline", IOW one that turns into a straight line (`.zeroVelocityAtStart` / `End` set to `false`). When starting and stopping, we want the spline to accelerate and decelerate, respectively (`.zeroVelocityAtStart` / `End` set to `true`).